fs_storage/models/fs_storage.py

```diff
@@ -116,6 +116,12 @@ def __init__(self, env, ids=(), prefetch_ids=()):
         compute="_compute_json_options",
         inverse="_inverse_json_options",
     )
+
+    eval_options_from_env = fields.Boolean(
```

Comment:
shouldn't be this flag controlled via server env settings?

Reply:
I'm just thinking that if you want the env conf to take full precedence, you should be able to set it via env key too otherwise the env settings are useless if you don't have the flag set.

However, I understand that you might want to leave the control from the UI.